com.unity.shadergraph/Editor/GraphDeltaRegistry/Registry/TextureType.cs

```diff
@@ -31,11 +30,21 @@ public ShaderFunction GetShaderFunction(NodeHandler node, ShaderContainer contai
             var builder = new ShaderFunction.Builder(container, GetRegistryKey().Name);
             var shaderType = registry.GetShaderType(node.GetPort(kTexture).GetTypeField(), container);
 
+            // Need to ignore the sampler if there is no user data on it or if it isn't connected, textures have one they are loaded w/already.
+            var samplerPort = node.GetPort(kSampler);
+            bool isConnected = samplerPort.GetConnectedPorts().Count() != 0;
+            bool isInitialized = SamplerStateType.IsInitialized(samplerPort.GetTypeField());
+            bool hasSampler = isConnected || isInitialized;
+
+            // hasSampler = true; // can enable to force usage of sampler for testing.
```

Comment:
When the SF bug is fixed, this could be turned on to force sampler state usage and check to see if things are functional. Proper tests will be added after this doesn't fail.